src/ripple/app/hook/Enum.h

```diff
@@ -340,6 +343,14 @@ enum ExitType : uint8_t {
     WASM_ERROR = 1,
     ROLLBACK = 2,
     ACCEPT = 3,
+    JSVM_ERROR = 4,
+    LEDGER_ERROR =
+        5,  // if the ledger contained for example a nonsense hookapi number
```

Comment:
If the Instruction Limit is reached, the ExitType is currently set to UNSET, but it may be worth adding a new ExitType(`INSTRUCTION_LIMIT_REACHED`) for this case.